sklbench/report/implementation.py

```diff
@@ -239,6 +239,7 @@ def get_result_tables_as_df(
     bench_cases = pd.DataFrame(
         [flatten_dict(bench_case) for bench_case in results["bench_cases"]]
     )
+    bench_cases = bench_cases.map(lambda x: str(x) if not isinstance(x, Hashable) else x)

```

Comment:
What is non-hashable object you are trying to convert?

Reply:
basic statistics result_options parameter is a list